redash/settings/__init__.py

```diff
@@ -259,6 +259,9 @@ def all_settings():
 # Frequency of schema samples refresh
 SCHEMA_SAMPLE_REFRESH_FREQUENCY_DAYS = int(os.environ.get("REDASH_SCHEMA_SAMPLE_REFRESH_FREQUENCY_DAYS", 14))
 
+# Frequency of schema samples refresh when no samples are stored
+SCHEMA_SAMPLE_EMPTY_REFRESH_FREQUENCY_DAYS = int(os.environ.get("REDASH_SCHEMA_SAMPLE_EMPTY_REFRESH_FREQUENCY_DAYS", 2))
```

Comment:
I'm not sure if the name really helps understand the purpose of the config value.

Could we maybe just rename `SCHEMA_SAMPLE_REFRESH_FREQUENCY_DAYS` to `SCHEMA_SAMPLE_**UPDATE**_FREQUENCY_DAYS` since the task is named `update_sample` as well?

Then we could name the new config value to be used in the `refresh_samples` tasks simply `SCHEMA_SAMPLE_REFRESH_FREQUENCY_DAYS`?